src/Common/Uuid/UuidRegistry.php

```diff
@@ -144,15 +147,64 @@ public function createUuid()
         return $uuid;
     }
 
+    // Generic function to update all missing uuids (to be primarily used in service that is run intermittently; should not use anywhere else)
+    // When add support for a new table uuid, need to add it here
+    //  Will log by default
+    public static function populateAllMissingUuids($log = true)
+    {
+        $logEntryComment = '';
+
+        // Standard updates
```

Comment:
need to add insurance_companies and insurance_data (and also optimize the index creation for these two in the 6.0.0 to 6.1.0 upgrade script)